CHANGES

```diff
@@ -1,3 +1,8 @@
+0.9.0 (unreleased)
+======================
+
+- fix loading many-to-many relationships (via secondary)
```

Comment:
"fix" implies it was broken. But it was just not supported... this is a new feature, right?